omise/controllers/front/payment.php

```diff
@@ -0,0 +1,43 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+    require_once _PS_MODULE_DIR_ . '/omise/classes/payment_order.php';
+}
+
+class OmisePaymentModuleFrontController extends ModuleFrontController
+{
+    public $display_column_left = false;
+    protected $payment_order;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->setPaymentOrder(new PaymentOrder());
```

Comment:
@nimid ✋ have question
At the `initContent()` method you create new `Charge` instance there but for `PaymentOrder` instance you chose to assign to a class's property here.

Why not be something like,
```php
public function __construct()
{
    $this->setPaymentOrder(new PaymentOrder())
    $this->setCharge(new Charge())
}
```

or
```php
public function initContent()
{
    $paymentOrder = new PaymentOrder();
    $omiseCharge = new Charge();
}
```